modules/extensions-mule-support/src/main/java/org/mule/runtime/module/extension/mule/internal/loader/parser/AbstractMuleSdkExtensionModelParser.java

```diff
@@ -6,64 +6,56 @@
+import static org.mule.runtime.extension.api.dsl.syntax.DslSyntaxUtils.getSanitizedElementName;
+
 import static java.util.Collections.emptyList;
 import static java.util.Collections.emptyMap;
 import static java.util.Optional.empty;
-import static java.util.Optional.of;
 import static java.util.function.Function.identity;
 import static java.util.stream.Collectors.toMap;
-import static org.mule.runtime.api.component.TypedComponentIdentifier.ComponentType.OPERATION_DEF;
-import static org.mule.runtime.api.meta.Category.COMMUNITY;
-import static org.mule.runtime.extension.api.dsl.syntax.DslSyntaxUtils.getSanitizedElementName;
-import static org.mule.runtime.internal.dsl.DslConstants.THIS_NAMESPACE;
-import static org.mule.runtime.internal.dsl.DslConstants.THIS_PREFIX;
-import static org.mule.sdk.api.annotation.Extension.MULESOFT;
 
 import org.mule.metadata.api.TypeLoader;
 import org.mule.metadata.api.model.MetadataType;
-import org.mule.runtime.api.meta.Category;
 import org.mule.runtime.api.meta.model.ExternalLibraryModel;
 import org.mule.runtime.api.meta.model.ModelProperty;
 import org.mule.runtime.api.meta.model.deprecated.DeprecationModel;
 import org.mule.runtime.api.meta.model.notification.NotificationModel;
 import org.mule.runtime.ast.api.ArtifactAst;
+import org.mule.runtime.ast.api.ComponentAst;
 import org.mule.runtime.ast.internal.model.ExtensionModelHelper;
 import org.mule.runtime.extension.api.property.SinceMuleVersionModelProperty;
 import org.mule.runtime.module.extension.internal.loader.java.property.ExceptionHandlerModelProperty;
-import org.mule.runtime.module.extension.internal.loader.java.property.LicenseModelProperty;
 import org.mule.runtime.module.extension.internal.loader.parser.ConfigurationModelParser;
 import org.mule.runtime.module.extension.internal.loader.parser.ConnectionProviderModelParser;
 import org.mule.runtime.module.extension.internal.loader.parser.ErrorModelParser;
 import org.mule.runtime.module.extension.internal.loader.parser.ExtensionModelParser;
 import org.mule.runtime.module.extension.internal.loader.parser.FunctionModelParser;
 import org.mule.runtime.module.extension.internal.loader.parser.OperationModelParser;
 import org.mule.runtime.module.extension.internal.loader.parser.SourceModelParser;
-import org.mule.runtime.module.extension.internal.loader.parser.XmlDslConfiguration;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.stream.Stream;
 
 /**
  * {@link ExtensionModelParser} implementation for Mule SDK extensions
  *
  * @since 4.5.0
  */
-public class MuleSdkExtensionModelParser implements ExtensionModelParser {
-
-  // The namespace of the extension when it's defined within an application rather than in a separate artifact.
-  public static final String APP_LOCAL_EXTENSION_NAMESPACE = "THIS";
+public abstract class AbstractMuleSdkExtensionModelParser extends BaseMuleSdkExtensionModelParser
```

Comment:
Abstract extends Base? Rename one of the two

Reply:
Renamed the abstract one to just `MuleSdkExtensionModelParser`.